testament/categories.nim

```diff
@@ -671,10 +670,6 @@ proc processCategory(r: var TResults, cat: Category,
                      options, testsDir: string,
                      runJoinableTests: bool) =
   case cat.string.normalize
-  of "rodfiles":
-    when false:
-      compileRodFiles(r, cat, options)
-      runRodFiles(r, cat, options)
```

Comment:
These are related to IC afaik. They are just not tested here, but is probably used for development elsewhere. I'd say you should keep the code.

/cc @disruptek since he's working on this.

Reply:
these procs don't even exist anymore